pyzo/util/interpreters/__init__.py

```diff
@@ -175,11 +175,12 @@ def _get_interpreters_conda():
     exes = []
     filename = os.path.expanduser("~/.conda/environments.txt")
     if os.path.isfile(filename):
-        for line in open(filename, "rt").readlines():
-            line = line.strip()
-            exe_filename = os.path.join(line, pythonname)
-            if line and os.path.isfile(exe_filename):
-                exes.append(exe_filename)
+        with open(filename, "rt") as fd:
```

Comment:
```suggestion
        with open(filename, "rt", encoding="utf-8") as fd:
```

Reply:
Good catch! I was not sure about the encoding used by conda's environments.txt so I did not touch this.
Looking into that it turns out that on older conda versions the native text encoding of the operating system was used and in newer conda versions utf-8, see https://github.com/conda/conda/issues/8662